admin_setting_twilio.go

```diff
@@ -7,7 +7,7 @@ import (
 // Compile-time proof of interface implementation.
 var _ TwilioSettings = (*adminTwilioSettings)(nil)
 
-// TwilioSettings describes all the Twilio admin settings.
+// TwilioSettings describes all the Twilio admin settings for the Admin Setting API https://www.terraform.io/cloud-docs/api-docs/admin/settings.
```

Comment:
So I'm curious if we should move the link to a new line for more readability.

Reply:
Will do.